workflows/crisprseq.nf

```diff
@@ -71,16 +97,329 @@ workflow CRISPRSEQ {
     INPUT_CHECK (
         ch_input
     )
+    .reads
+    .map {
+        meta, fastq ->
+            def meta_clone = meta.clone()
+            meta_clone.id = meta_clone.id.split('_')[0..-2].join('_')
+            [ meta_clone, fastq ]
+    }
```

Comment:
```
            meta_clone.id = meta.id.split('_')[0..-2].join('_')
```
Shouldn't you want to get the meta.id here?
But following @robsyme advance training, if I'm not mistaken, we can actually get rid of the meta.clone() logic, and use something like that:
```
    .map{ meta, fastq -> [ meta - meta.subMap('id') + [id: meta.id.split('_')[0..-2].join('_')], fastq ] }
```
cf https://github.com/nf-core/sarek/pull/896